aws/lambda/python/log-anonymiser/module.tf

```diff
@@ -0,0 +1,42 @@
+data "aws_iam_role" "lambda_role" {
+  role_name = "lambda"
```

Comment:
can we terraform this role, or use a terraformed existing role rather than `data`? I know we use this in some places but only as a workaround where a terraform defined role is not possible.

Reply:
I think there's no reason not to, but the existing lambdas are using the "lambda" role. It would be good to chat about how best to do this and maybe pair on it later this week?